Formula/httpd.rb

```diff
@@ -0,0 +1,132 @@
+class Httpd < Formula
+  desc "HTTP server"
+  homepage "https://httpd.apache.org/"
+  url "https://archive.apache.org/dist/httpd/httpd-2.4.27.tar.bz2"
+  sha256 "71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a"
+
+  bottle do
+    sha256 "d35e67ae745053cda23273a520ead6bde506ec8082057edcd3185c9e36eae483" => :sierra
+    sha256 "4ea7056d6c08f9cf49507869aa8c78e3522c601d87b3347d31561ba05b33d41c" => :el_capitan
+    sha256 "e6ae5c4d38b40bcf3d85877ac117a1ea048de774edec3c2b0ff714bc01a6d40a" => :yosemite
+  end
+
+  skip_clean :la
+
+  deprecated_option "with-http2" => "with-nghttp2"
+
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "apr-util"
```

Comment:
Should explicitly declare `depends_on "apr"` too since we reference it below with `--with-apr=#{Formula["apr"].opt_prefix}`